monitors/rootmonitor/src/ROOTMonitor.cc

```diff
@@ -0,0 +1,132 @@
+#include "eudaq/ROOTMonitor.hh"
+#include "eudaq/FileDeserializer.hh"
+#include "eudaq/FileReader.hh"
+#include "eudaq/DataConverter.hh"
+
+#include "TH1.h"
+#include "TGraph.h"
+
+#include <ratio>
+#include <chrono>
+#include <thread>
+
+namespace eudaq {
+  ROOTMonitor::ROOTMonitor(const std::string & name, const std::string & title, const std::string & runcontrol)
+    :Monitor(name, runcontrol),
+     m_app(new TApplication(name.c_str(), nullptr, nullptr)),
+     m_monitor(new ROOTMonitorWindow(m_app.get(), title)){
+    if (!m_monitor)
+      EUDAQ_THROW("Error allocating main window");
+
+    m_monitor->SetStatus(ROOTMonitorWindow::Status::idle);
```

Comment:
A new instance of any EUDAQ component will start up automatically in an 'uninit' state. I do not think you'll need that state here.

Reply:
Would it make sense to add a `eudaq::Status::State` accessor to the `eudaq::CommandReceiver` API, or possibly turn `m_status` into a protected member?

Reply:
@lhuth Any strong feeling on this?

Reply:
I personally do not have a strong opinion on that. However, @eyiliu is the core-expert and maybe has strong feelings on it.

Reply:
Thanks, @lhuth !